engine/CMakeLists.txt

```diff
@@ -629,537 +678,561 @@ CONFIGURE_FILE(${Vega_Strike_SOURCE_DIR}/cmake-config.h.in ${Vega_Strike_BINARY_
-    src/gfx/lerp.cpp
-    src/gfx/matrix.cpp
-    src/gfx/mesh_bxm.cpp
-    src/gfx/mesh_poly.cpp
-    src/gfx/mesh_xml.cpp
-    src/gfx/mesh.cpp
-    src/gfx/quaternion.cpp
-    src/gfx/soundcontainer_generic.cpp
-    src/gfx/sphere_generic.cpp
-    src/gfx/tvector.cpp
-)
+        src/gfx/texture_manager.cpp
+        src/gfx/boltdrawmanager.cpp
+        src/gfx/cockpit_generic.cpp
+        src/gfx/lerp.cpp
+        src/gfx/matrix.cpp
+        src/gfx/mesh_bxm.cpp
+        src/gfx/mesh_poly.cpp
+        src/gfx/mesh_xml.cpp
+        src/gfx/mesh.cpp
+        src/gfx/quaternion.cpp
+        src/gfx/soundcontainer_generic.cpp
+        src/gfx/sphere_generic.cpp
+        src/gfx/tvector.cpp
+        )
 
 SET(LIBGFXCLIENT_SOURCES
-    src/gfx/ani_texture.cpp
-    src/gfx/animation.cpp
-    src/gfx/aux_logo.cpp
-    src/gfx/aux_palette.cpp
-    src/gfx/aux_texture.cpp
-    src/gfx/background.cpp
-    src/gfx/camera.cpp
-    src/gfx/cockpit_xml.cpp
-    src/gfx/cockpit.cpp
-    src/gfx/cockpit_gfx.cpp
-    src/gfx/cockpit_gfx_utils.cpp
-    src/gfx/coord_select.cpp
-    src/gfx/env_map_gent.cpp
-    src/gfx/gauge.cpp
-    src/gfx/halo_system.cpp
-    src/gfx/halo.cpp
-    src/gfx/hud.cpp
-    src/gfx/jpeg_memory.cpp
-    src/gfx/loc_select.cpp
-    src/gfx/masks.cpp
-    src/gfx/mesh_bin.cpp
-    src/gfx/mesh_fx.cpp
-    src/gfx/mesh_gfx.cpp
-    src/gfx/nav/criteria_xml.cpp
-    src/gfx/nav/criteria.cpp
-    src/gfx/nav/drawgalaxy.cpp
-    src/gfx/nav/drawlist.cpp
-    src/gfx/nav/drawsystem.cpp
-    src/gfx/nav/navcomputer.cpp
-    src/gfx/nav/navgetxmldata.cpp
-    src/gfx/nav/navpath.cpp
-    src/gfx/nav/navscreen.cpp
-    src/gfx/nav/navscreenoccupied.cpp
-    src/gfx/radar/bubble_display.cpp
-    src/gfx/radar/bubble_display.h
-    src/gfx/radar/dual_display.cpp
-    src/gfx/radar/dual_display.h
-    src/gfx/radar/null_display.h
-    src/gfx/radar/plane_display.cpp
-    src/gfx/radar/plane_display.h
-    src/gfx/radar/radar.cpp
-    src/gfx/radar/radar.h
-    src/gfx/radar/sensor.cpp
-    src/gfx/radar/sensor.h
-    src/gfx/radar/sphere_display.cpp
-    src/gfx/radar/sphere_display.h
-    src/gfx/radar/track.cpp
-    src/gfx/radar/track.h
-    src/gfx/radar/viewarea.cpp
-    src/gfx/radar/viewarea.h
-    src/gfx/particle.cpp
-    src/gfx/pipelined_texture.cpp
-    src/gfx/quadsquare_cull.cpp
-    src/gfx/quadsquare_render.cpp
-    src/gfx/quadsquare_update.cpp
-    src/gfx/quadsquare.cpp
-    src/gfx/quadtree_xml.cpp
-    src/gfx/quadtree.cpp
-    src/gfx/ring.cpp
-    src/gfx/occlusion.cpp
-    src/gfx/screenshot.cpp
-    src/gfx/soundcontainer.cpp
-    src/gfx/soundcontainer_aldrv.cpp
-    src/gfx/sphere.cpp
-    src/gfx/sprite.cpp
-    src/gfx/star.cpp
-    src/gfx/stream_texture.cpp
-    src/gfx/technique.cpp
-    src/gfx/pass.cpp
-    src/gfx/tex_transform.cpp
-    src/gfx/vdu.cpp
-    src/gfx/vid_file.cpp
-    src/ffmpeg_init.cpp
-    src/gfx/vsbox.cpp
-    src/gfx/vsimage.cpp
-    src/gfx/warptrail.cpp
-)
+        src/gfx/ani_texture.cpp
```

Comment:
Easier to have these cosmetic (tabs and copyright changes) in a separate PR. It's pretty much auto approve and removes a lot of noise from here.
Also, probably a good idea to squash the commits to keep the tree cleaner.

Reply:
I can squash the commits when merging this PR. I am fine with that.

Do you want me to reverse these cosmetic changes to CMakeLists.txt in this PR, and then reformat later in a separate PR?

Reply:
No. It's already in review and I'm not that strict about full separation. Everything in moderation.